Source/Renderer/Context.js

```diff
@@ -268,13 +268,17 @@ define([
         this._antialias = gl.getContextAttributes().antialias;
 
         // Query and initialize extensions
-        this._standardDerivatives = !!getExtension(gl, ['OES_standard_derivatives']);
-        this._elementIndexUint = !!getExtension(gl, ['OES_element_index_uint']);
-        this._depthTexture = !!getExtension(gl, ['WEBGL_depth_texture', 'WEBKIT_WEBGL_depth_texture']);
-        this._textureFloat = !!getExtension(gl, ['OES_texture_float']);
-        this._fragDepth = !!getExtension(gl, ['EXT_frag_depth']);
+        this._standardDerivatives = !!getExtension(gl, ['OES_standard_derivatives']) || this._webgl2;
+        this._elementIndexUint = !!getExtension(gl, ['OES_element_index_uint']) || this._webgl2;
+        this._depthTexture = !!getExtension(gl, ['WEBGL_depth_texture', 'WEBKIT_WEBGL_depth_texture']) || this._webgl2;
+        this._textureFloat = !!getExtension(gl, ['OES_texture_float']) || this._webgl2;
+        this._fragDepth = !!getExtension(gl, ['EXT_frag_depth']) || this._webgl2;
         this._debugShaders = getExtension(gl, ['WEBGL_debug_shaders']);
 
+        if (this._webgl2) {
+            this._colorBufferFloat = !!getExtension(gl, ['EXT_color_buffer_float']);
+        }
```

Comment:
Actually @bagnell's suggestion is better: https://github.com/AnalyticalGraphicsInc/cesium/pull/5568#discussion_r124891493